### DISCUSSION TITLE
How to controll translate props

### DISCUSSION THREAD

**Discussion by p4rtridge** [davidjerleke/embla-carousel#604] [Help]
Is it possible to controll translate3d props? I want to make wheel gesture but stop momentun when I stop scrolled

**Comment by davidjerleke** [comment]
Hi @partridge1307,

I'm not sure what the question is? Would you mind giving me more details?

**Reply by p4rtridge** [reply]
I want to use wheel gestures to make it like normal scroll. But when I stop scrolling it still have some momentum. I want to force scroll slide when I scrolling and stop scroll right away when I stop scrolling.

**Reply by davidjerleke** [reply]
@partridge1307 have you noticed that there's a [plugin](https://www.embla-carousel.com/plugins/wheel-gestures/) for wheel gestures?

Best,
David

**Reply by davidjerleke** [reply]
However, if you want to scroll instantly, you can use [this approach](https://github.com/davidjerleke/embla-carousel/discussions/602#discussioncomment-7355729).
